packages/react/src/components/form-popup/FormPopup.tsx

```diff
@@ -1,50 +1,59 @@
-import React from 'react';
+import React, { useRef } from 'react';
 
 import { Popup, ToolbarItem } from 'devextreme-react/popup';
+import ValidationGroup from 'devextreme-react/validation-group';
 import { useScreenSize } from './../../utils/media-query';
 
 type PopupProps = {
   title: string,
   visible: boolean,
   changeVisibility: () => void,
-  onSaveClick: () => void,
 }
 
-export const FormPopup = ({ title, visible, changeVisibility, onSaveClick, children } : React.PropsWithChildren<PopupProps>) => {
+export const FormPopup = ({ title, visible, changeVisibility, children }: React.PropsWithChildren<PopupProps>) => {
   const { isXSmall, isSmall } = useScreenSize();
+  const validationGroup = useRef<ValidationGroup>(null);
+
+  const onSaveClick = () => {
+    if (!validationGroup.current?.instance.validate().isValid) return;
```

Comment:
Why is there using ref instead of `e.validationGroup.validate().isValid` ?

Reply:
Validation doesn't work if open popup at the first time.. The same problem was with file uploader.